.github/dependabot.yml

```diff
@@ -1,5 +1,83 @@
 version: 2
 updates:
+  - package-ecosystem: npm
```

Comment:
nit: I'd place the NPM packages between the "cargo" and "github-actions" blocks.